build/charts/flow-aggregator/conf/flow-aggregator.conf

```diff
@@ -187,3 +187,7 @@ flowLogger:
   # PrettyPrint enables conversion of some numeric fields to a more meaningful string
   # representation.
   prettyPrint: {{ .Values.flowLogger.prettyPrint }}
+
+# Provide a clusterID to be added to records. By default this # ID is an autogenerated UUID which
```

Comment:
the change is wrong, you have a random `#` sign in the middle of your comment line